plugins/inventory/centurion.py

```diff
@@ -1,5 +1,6 @@
 import json
```

Comment:
why are you editing this file?

Reply:
I didn't edit that at all

Reply:
if it shows in a PR, you edited it. I'm guessing you fucked the commits somewhere

Reply:
I'm looking at that right now...... Why the fuck is that showing up?? I only staged each of the playbooks and pushed them seperately. Didn't touch any of this other stuff

Reply:
I'm guessing when you did a git reset you went back to the wrong commit.

I'd reset back to the dev common commit and manually re-add the commits back.

Reply:
Ack

Reply:
That looks better

Reply:
indeed